go.mod

```diff
@@ -646,3 +646,391 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 	skywalking.apache.org/repo/goapi v0.0.0-20220121092418-9c455d0dda3f // indirect
 )
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension => ../github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension
```

Comment:
This probably derailed CI.

Reply:
This is mystery for me, why are here any changes at all.

Reply:
I have replaced it with version from main. :/